### DISCUSSION TITLE
Premier League Stadiums

### DISCUSSION THREAD

**Discussion by jacobf19** [jakobzhao/geog495#51] [Final Project]
***Link:***https://jacobf19.github.io/eplstadiums/
***GitHub:*** https://github.com/jacobf19/eplstadiums
This is a web map which displays the stadiums of the English Premier League soccer league. This map sorts distance to the nearest stadium as well as displaying facts about the stadiums.
![fullwebpage](https://user-images.githubusercontent.com/72268448/145458541-5f0ac45d-ff96-4823-af8d-f83966abbc50.png)

**Comment by VaneRaklan** [comment]
I really like the design, change of color palette for the sidebar and the icons (specifically the size) since it's very easy to see them against the base map!
You're already providing some good information in the popup and the side panel, but I think something interesting that you could include would be team data. It could be as little as the team's current W-L, their standings compared to other teams, etc.

**Comment by kevography** [comment]
I like how much information you supply about the stadiums in both the sidebar and the popups. The map is very legible and it is really interesting to see how the stadiums are distributed in the country. 

One added functionality that I think would be really helpful is if you added a fly-to function that activated when we click on the stadiums in the sidebar. It feels a little misleading to have the names react when hovered over but not respond when clicked on.

**Comment by zhuxiaoyu1019** [comment]
It is an intuitive map illustrating the premier league stadiums in the United Kingdom. UI interaction is easy and friendly. The pop-up information is very helpful.
There's one thing I noticed: Your title link isn't working. The other suggestion is that since your data has numerical variables, it might be helpful if you implement a sorting mechanism based on them to make it easier for users to find stadiums based on their queries.

**Comment by notzulkifli** [comment]
I like the interactivity of your map! I found that once i typed in a location of a soccer league stadium, the miles and distance changes which is really neat! It shows great use of data sorting!

One possible thing to add is the ability to click on the side panel on the left and have it maneuver the the map to that specific spot. For example when clicking on "Arsenal FC" instead of having to just type it, we can click on the name on the left side-panel and have it locate us to the point!

**Comment by seanhy1** [comment]
This user interface is very clean and polished. The interactivity was quite smooth, and all in all, the basemap is quite pleasant to the eye in my opinion. The choice of customizing your marker points to the EPL logo was a nice touch as well. Soccer is one of my favorite sports so seeing a web application revolving around such topic was refreshing to see. 

One change that isn't necessary, but would touch up the design of your map is implementing a way to create various marker points based on the team of which is hosted by each EPL stadium. Again, this isn't a change that is NEEDED, but it would be pretty cool to see some more customization. All in all, well done Jacob and COYS!

**Comment by manhongg** [comment]
I really liked the simplistic readability and minimalistic approach to your web app! The concept and data that you chose was a very unique but interesting concept for the final project, and I felt like your explanation of the usability and purpose of the project was thoroughly explained and interesting to read. The base map that you used was also aesthetically pleasing and it conveyed the location data well. One idea for improvement would be to add more filters, as this would make the app more interactive and allow the user to engage with the data on a deeper level. Overall, great job on your project!

**Comment by nlertv** [comment]
I really like your website and how you include all of the details about each stadium. Providing information on the sidebar and having the ability to search on the geocoder is very helpful. The design of the webpage also looks very clean and organized as well. I think to improve, it would be nice to be able to move the pinned location to other cities and compare the distance of the stadiums to that city.

**Comment by TheAdonalsium** [comment]
I really like the additional details you included about each stadium. If I was to suggest one thing to improve, it would be a slightly smaller zoom, so that the clusters in the larger cities are more spread out.

**Comment by Subarashiihibi** [comment]
I like your user interface  of your application. The content in side bar is really distinctive and useful for users to track different information. I also like the image for your markers which makes your map characteristic. I think you could make your pop-ups closable so the users could do multiple queries without refreshing the web page.
